proto/data.proto

```diff
@@ -129,11 +129,19 @@ message ChangeReplicasTrigger {
       (gogoproto.customname) = "NextReplicaID", (gogoproto.casttype) = "ReplicaID"];
 }
 
+// ModifiedSpanTrigger indicates that a specific span has been modified.
+// This can be used to trigger scan-and-gossip for the given span.
+message ModifiedSpanTrigger {
+  optional bool system_db_span = 1 [(gogoproto.nullable) = false, (gogoproto.customname) = "SystemDBSpan"];
+}
+
 // CommitTrigger encapsulates all of the internal-only commit triggers.
+// Only one may be set.
```

Comment:
I'm guessing the "only one of" in the implementation is because there's no way currently for more than one of those to happen at the same time. Enforcing it through the proto is a bit premature, this comment was just about clarification.